core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -648,6 +649,8 @@ public Map<String, SaltPillarProperties> createPillarWithClouderaManagerSettings
                                 "missed_heartbeat_interval", cmMissedHeartbeatInterval,
                                 "disable_auto_bundle_collection", disableAutoBundleCollection,
                                 "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
+                                "is_version_greater_than_762", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_6_2),
```

Comment:
I think it would be better instead of pointing to a version we should point to a feature like isApplicableForCloudProviderSetup

Reply:
@satyabollineni dont you want to change this as I highlighted?